hipe4ml/tree_handler.py

```diff
@@ -145,6 +172,28 @@ def apply_preselections(self, preselections, inplace=True, **kwds):
         self._preselections = preselections
         return self._full_data_frame.query(preselections, inplace=inplace, **kwds)
 
+    def apply_model_handler(self, model_handler, output_margin=True):
```

Comment:
Let's add an optional parameter with the name of the new column, that might be useful